core/src/test/java/com/novoda/noplayer/mediaplayer/ExceptionMatcher.java

```diff
@@ -0,0 +1,28 @@
+package com.novoda.noplayer.mediaplayer;
+
+import org.hamcrest.BaseMatcher;
+import org.hamcrest.Description;
+
+class ExceptionMatcher extends BaseMatcher<Exception> {
```

Comment:
based on the `matches`, this is a `ExceptionMessageMatcher` or do you think it's implied by the parameters?